docs: fix broken NgZone code example

juristr opened the pull request:
The current code example was broken as there were a couple of syntax errors. This commit fixes the demo.

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)


## PR Type
What kind of change does this PR introduce?

<!-- Please check the one that applies to this PR using "x". -->
```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] angular.io application / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?
<!-- Please describe the current behavior that you are modifying, or link to a relevant issue. -->
The current inline code example that contains a demo for NgZone was broken, containing a couple of syntax errors. Found it when copy & pasting it from the angular.io docs.

Issue Number: N/A

## What is the new behavior?

This commit fixes the demo example. Tried it here: https://stackblitz.com/edit/angular-ngzones-demo

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

<!-- If this PR contains a breaking change, please describe the impact and migration path for existing applications below. -->


## Other information

juristr committed:
docs(core): fix broken NgZone code example

The current code example was broken as there were a couple of syntax errors. This commit fixes the demo.

juristr commented:
@wardbell sorry for poking you directly. Could you or someone give a quick look at this. It's an easy fix of a code example which is wrong on the documentation site.

The build fails, but doesn't seem related to my change. Is there anything I need to update or rebase to make it pass?

thx :)

gkalpak commented on packages/core/src/zone/ng_zone.ts:
```diff
@@ -63,17 +63,18 @@ import {EventEmitter} from '../event_emitter';
- *       this._ngZone.run(() => {console.log('Outside Done!') });
- *     }}));
+ *         // reenter the Angular zone and display done
+ *         this._ngZone.run(() => {console.log('Outside Done!') });
```
While you're at it, would you like to also fix:

`{console.log('Ouside Done!') }` --> `{ console.log('Ouside Done!'); }`

juristr replied:
Oh, right, sure 😄, but without introducing again the "Ouside.." typo 😉. Will do immediately

mary-poppins commented:
You can preview 24a18e5 at https://pr19291-24a18e5.ngbuilds.io/.

juristr commented:
@mary-poppins Oh cool, didn't know, very helpful. Just verified and my corrected example runs properly :)

gkalpak reviewed (changes requested):
Ideally, this should not be hardcoded, but come from an actual running [example app][1] (just in case you feel adventurous) :smiley:

Can you please rebase on latest master and change the commit message to `docs(core): ...`?

Other than that LGTM (as soon as Travis is happy).

[1]: https://github.com/angular/angular/tree/master/packages/examples/core

juristr commented:
> Ideally, this should not be hardcoded, but come from an actual running example app (just in case you feel adventurous) 😃

Aha, I see what you mean. Ya for sure. I'll give it a look whether I can convert it to a runnable example. I'll ping you back in case I'm stuck 😄

mary-poppins commented:
You can preview 6705751 at https://pr19291-6705751.ngbuilds.io/.

juristr commented:
@gkalpak Ok, rebased and updated my code. Checks seem to pass now properly. For now this should work and be ready to merge. I'll check the conversion into a runnable example later. Will come back with another PR, promised :)

gkalpak commented on packages/core/src/zone/ng_zone.ts:
```diff
@@ -63,17 +63,18 @@ import {EventEmitter} from '../event_emitter';
- *       this._ngZone.run(() => {console.log('Outside Done!') });
- *     }}));
+ *         // reenter the Angular zone and display done
+ *         this._ngZone.run(() => {console.log('Outside Done!'); });
```
Nit there could be a space before `console` :grin:

juristr replied:
🙄 😉 
If ur ok with it, I'd leave it like this and fix it when I convert it to an executable example, as discussed earlier.

gkalpak replied:
Oh, look, it's fixed :wink:

gkalpak reviewed (approved):
LGTM :+1: (but still needs approval from `core` owners :grin:)

gkalpak commented:
@juristr, I had hoped for an observable of PRs, but a promise will do for now :stuck_out_tongue:

mary-poppins commented:
You can preview 9eb34c9 at https://pr19291-9eb34c9.ngbuilds.io/.
